run_checks.php

```diff
@@ -151,6 +159,20 @@
     exit(1);
 }
 
+// Check that old events are still match expected format
+try {
+    list($result_old) = $client->searchEvents(1, start: $start->getTimestamp() * 1000, end: $end->getTimestamp() * 1000, reverse: true);
+    $identification_data_old = $result_old->getEvents()[0]->getProducts()->getIdentification()->getData();
+    $visitor_id_old = $identification_data_old->getVisitorId();
+    $request_id_old = $identification_data_old->getRequestId();
+
+    list($result, $response) = $client->getEvent($request_id_old);
+    list($result, $response) = $client->getVisits($visitor_id_old);
+    fwrite(STDERR, sprintf("\n\nOld events are good\n"));
+}  catch (Exception $e) {
+    fwrite(STDERR, sprintf("\n\nException when trying to read old data: %s\n", $e->getMessage()));
```

Comment:
In both `$response`, we could do additional checks, such as comparing `visitorId` and `requestId` to `$visitor_id_old` and `$request_id_old` to be sure that both models were instanced correctly.

Reply:
Added!